css/pages/review.scss

```diff
@@ -0,0 +1,70 @@
+// Review
+.page {}
+
+.page--review {
+  color: #404040;
+
+  .page--review__title {
+    text-align: center;
+    font-style: bold;
+    font-weight: 700;
+    font-size: 2.5em;
+    letter-spacing: 3px;
+    margin: 40px;
+  }
+
+
+  .page--review__content {
+    width: 80%;
+    margin: 0 auto;
+
+    div {
+      display: inline-block;
+    }
+
+    .page--review__col--1 {
+      font-size: 1.5em;
+      width: 30%;
+      color: green;
+      vertical-align: top;
+    }
+
+    @media (max-width: $s__breakpoint--large) {
```

Comment:
Since this media query only changes `col--1`, you can actually put it inside the styles for `col--1` so you don't have to retype it.  For example:

```scss
.review__col--1 {
  // styles
  
  @media (max-width: $s__breakpoint--large) {
    // media query styles
  }
}
```